base_site/index.html

```diff
@@ -57,78 +69,91 @@ <h1 class="heading-style-h1 text-color-gray">Technical Documentation</h1>
                   <div class="product-list_content">
-                    <div class="product_logo-wrap"><img src="images/inquanto.svg" loading="lazy" alt="" class="image-fit-contain"></div>
+                    <a href="https://docs.quantinuum.com/tket/" class="product_logo-wrap is-tket w-inline-block"><img src="_static/images/tket.svg" loading="lazy" alt="" class="image-fit-contain"></a>
                     <div class="spacer-small"></div>
-                    <p class="product_description">State-of-the-art quantum chemistry platform for complex molecular and materials simulations</p>
+                    <p class="product_description">Hardware agnostic quantum compiler</p>
                     <div class="spacer-medium"></div>
-                    <a href="https://docs.quantinuum.com/inquanto/manual/howto.html" class="product_link-wrap w-inline-block">
-                      <p class="product_link-text">Get Started with InQuanto</p>
-                      <p class="product_description is-list">Learn how to accelerate quantum  computational chemistry</p>
+                    <a href="https://docs.quantinuum.com/tket/user-guide/" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Get Started with TKET</p>
+                      <p class="product_description is-list">Getting started tutorial showing basic usage of pytket</p>
+                    </a>
+                    <div class="spacer-small"></div>
+                    <a href="https://docs.quantinuum.com/tket" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Documentation for TKET</p>
+                      <p class="product_description is-list">Overview of all TKET documentation including the user guide, API docs, and developer blog</p>
                     </a>
+                  </div>
+                </div>
+                <div id="w-node-_47d38466-c01a-d6de-3e3d-1a087be788d1-c2a09c1d" class="w-layout-grid product-list_component">
+                  <div class="product-list_content">
+                    <a href="https://docs.quantinuum.com/lambeq/" class="product_logo-wrap is-lambeq w-inline-block"><img src="_static/images/Quantinuum-Products_Lambeq_black.png" loading="lazy" sizes="128px" srcset="_static/images/Quantinuum-Products_Lambeq_black-p-500.png 500w, _static/images/Quantinuum-Products_Lambeq_black.png 947w" alt="" class="image-fit-contain"></a>
                     <div class="spacer-small"></div>
-                    <a href="https://docs.quantinuum.com/inquanto/index.html" class="product_link-wrap w-inline-block">
+                    <p class="product_description">Α Python toolkit for quantum natural language processing</p>
+                    <div class="spacer-medium"></div>
+                    <a href="http://docs.quantinuum.com/lambeq/intro.html" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Get Started with λambeq</p>
```

Comment:
Are we sure on "λambeq" here?

Reply:
```suggestion
                      <p class="product_link-text">Get Started with Lambeq</p>
```

Discussed with Irfan on Slack and we'll go with "Lambeq" for now. Can discuss this later.